ui/DesignSystem/Component/SourceBrowser/RevisionSelector.svelte

```diff
@@ -1,9 +1,13 @@
 <script>
-  import { createEventDispatcher } from "svelte";
+  import { createEventDispatcher, getContext } from "svelte";
+  import { push } from "svelte-spa-router";
+
+  import * as path from "../../../src/path.ts";
```

Comment:
We can cluster the `ts` imports.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/795/commits/b23b5b3135dd757c5fe4f566058dc012e401b910